internal/pkg/service/stream/source/httpsource/httpsource_test.go

```diff
@@ -58,7 +58,7 @@ func TestStart(t *testing.T) {
 	require.NoError(t, stream.StartComponents(ctx, d, mock.TestConfig(), stream.ComponentHTTPSource))
 
 	// Wait for the HTTP server
-	require.NoError(t, netutils.WaitForTCP(listenAddr, time.Second))
+	require.NoError(t, netutils.WaitForHTTP(url, 10*time.Second))
 	logger.AssertJSONMessages(t, `
 {"level":"info","message":"starting HTTP source node","component":"http-source"}
 {"level":"info","message":"started HTTP source on \"0.0.0.0:%d\"","component":"http-source"}
```

Comment:
Previous wait mechanism is not enough, for an unknown reason, maybe some `fasthttp` internal thing.

For standard Go HTTP server, the mechanism is OK: https://stackoverflow.com/a/53332769

![image](https://github.com/keboola/keboola-as-code/assets/19371734/0185329e-1415-403e-a445-84700c4856d7)